Makefile.am

```diff
@@ -87,9 +75,7 @@ libsass_la_SOURCES = \
 	utf8_string.cpp utf8_string.hpp \
 	util.cpp util.hpp
 
-libsass_la_CFLAGS = $(AM_CFLAGS)
-libsass_la_CXXFLAGS = $(AM_CXXFLAGS)
-libsass_la_LDFLAGS = $(AM_LDFLAGS) -no-undefined -version-info 0:9:0
+libsass_la_LDFLAGS = -no-undefined -version-info 0:9:0
```

Comment:
According to [this](http://www.gnu.org/software/automake/manual/html_node/Flag-Variables-Ordering.html):

> Automake always uses two of these variables when compiling C sources files. When compiling an object file for the mumble target, the first variable will be mumble_CPPFLAGS if it is defined, or AM_CPPFLAGS otherwise. The second variable is always CPPFLAGS.

So, you should still need `$(AM_LDFLAGS)` here.